src/cli/lineage.ts

```
@@ -633,6 +633,108 @@
+
+  if (options.verbose) {
+    console.log('\nDetails:');
+    lineages.forEach((l, i) => {
+      console.log(`  ${i + 1}. ${l.id.substring(0, 8)} - ${l.kind} (${l.status})`);
+    });
+  }
+
+  return statusCounts;
+}
+
+async function confirmDeletion(options: LineageCleanOptions, statusCounts: Record<string, number>): Promise<boolean> {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'confirmDeletion'. Consider to delete it or rename it.

'confirmDeletion' is defined here:


(lint/suspicious/noRedeclare)